lumicks/pylake/tests/data/mock_confocal.py

```diff
@@ -121,28 +126,19 @@ def generate_photon_count_line(line):
 
 
 class MockConfocalFile:
```

Comment:
Is this class necessary anymore?
How about just renaming the static functions:
`MockConfocalFile.from_image` -> `mock_confocal_from_image`
`MockConfocalFile.from_streams` -> `mock_confocal_`

Reply:
Not needed anymore indeed. It's a bad habit of mine to sometimes use classes as namespaces. I'll replace them with free functions. Doing this, I realized that this thing is actually not taking slices. It's taking raw numpy arrays, so I changed the second one to `mock_confocal_from_arrays`.